routes/HomePage.vue

```diff
@@ -8,23 +8,26 @@
       @delete-credential="$event.waitUntil(deleteCredential($event))"
       @filtered-profiles="filteredProfiles = $event"
       @filtered-credentials-loading="loadingFilteredCredentials = $event" />
+    <install-credential-handler-btn />
```

Comment:
I think we'll want some additional text that guide the user to press this button -- especially if they haven't done it before or this is their first log into the wallet. We might not be able to tell that easily now -- in which case please file an issue that says we should guide the user here to have them hit this button in the future once we can figure that out.